Source/WebKit/UIProcess/API/Cocoa/WKProcessPoolPrivate.h

```diff
@@ -191,3 +191,9 @@ WK_CLASS_AVAILABLE(macos(14.5), ios(17.5), visionos(1.2))
 + (NSArray<_WKProcessInfo *> *)_networkingProcessInfo WK_API_AVAILABLE(macos(14.5));
 + (NSArray<_WKProcessInfo *> *)_webContentProcessInfo WK_API_AVAILABLE(macos(14.5));
 @end
+
+@interface WKProcessPool (WKPrivateMac)
+
+- (void)_registerAdditionalFont:(NSString *)fontName WK_API_AVAILABLE(macos(WK_MAC_TBA));
```

Comment:
Should this SPI take an array, so that we don't need to send multiple IPC messages for multiple fonts?

Reply:
That is a good point! I'll change that.

Thanks for reviewing!